docs/data/data-grid/selection/selection.md

```diff
@@ -84,25 +84,89 @@ The following demo shows the prop in action:
 
 {{"demo": "ControlledSelectionServerPaginationGrid.js", "bg": "inline"}}
 
-## apiRef [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+## Cell selection [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
```

Comment:
**Outside of the scope of this PR (maybe for a follow-up)**

It might be better to systematically have premium/pro features on a dedicated page if the section is larger than the viewport. My fear is that a developer might land in the middle of the section using the search without being able to easily see that it's a Pro/Premium feature. With a dedicated page, we get a UX benefit, even in the middle of the page, we see:

<img width="307" alt="Screenshot 2022-11-10 at 00 27 59" src="https://user-images.githubusercontent.com/3165635/200963893-369a1daa-dfac-46fd-87f2-9cc02b3be861.png">

Reply:
Sounds great